assets/js/interactivity/index.js

```diff
@@ -1,15 +1,17 @@
 import registerDirectives from './directives';
 import { init } from './router';
-export { store } from './store';
+import { runStoreCallbacks } from './store';
+
 export { navigate } from './router';
-import './cart/cart-store';
+export { store } from './store';
 
 /**
  * Initialize the Interactivity API.
  */
 document.addEventListener( 'DOMContentLoaded', async () => {
 	registerDirectives();
 	await init();
+	runStoreCallbacks( 'afterLoad' );
```

Comment:
I'm unsure if it is better to call `runStoreCallbacks` before or after `init`. I opted for the second option, but we can change it if you feel appropriate.

cc: @luisherranz

Reply:
It's correct. Before `init` is reserved for `beforeLoad`:

```js
await runStoreCallbacks( 'beforeLoad' );
await init();
runStoreCallbacks( 'afterLoad' );
```

Reply:
Maybe we should rename `init` to `load` 😆